.github/workflows/e2e-ci.yml

```diff
@@ -38,7 +38,11 @@ jobs:
         run: pnpm exec playwright install
 
       - name: Run Playwright E2E tests
-        run: pnpm test:e2e
+        # Due to a bug in pnpm on Windows this --filter flag does not work when
+        # called directly with a package.json script.
+        # In order to make it working we need to call it directly with pnpm in
+        # the CI command.
+        run: pnpm --filter='fixture-*' run test:e2e
```

Comment:
Is there an open issue in the pnpm repository that can be referenced?

Reply:
I found a bunch of threads online but haven't checked opened issues. People were just doing this workaround